### ISSUE TITLE
Some G38.2 Command Ignored

### ISSUE THREAD

**Issue by BusbyGT** [winder/Universal-G-Code-Sender#769]
Simple 3 axis probe script. When loaded, the file isn't fully loaded in the UI (no popup on Platform indicating an issue but the "Rows in File" stays at 0) but when streamed anyways, it stops before executing the "G38.2 Z-1 F2" command. The other 2 G38.2 commands are run but the third isn't. I want to say there is an error popup but I can't remember offhand.

Rpi 3 w/ Raspbian
GRBL v1.1f
Sept 10, 2017 nightly

[Kodiak-129644.txt](https://github.com/winder/Universal-G-Code-Sender/files/1312895/Kodiak-129644.txt)

**Comment by BusbyGT** [comment]
The original description/image in #727 describes what I am seeing if I recall correctly. Notice how in the image it barfs on the third G38.2 command? I think that is the same error that I get when streaming it as well.

**Comment by winder** [comment]
This is a bug, I agree it is a duplicate of #727

**Comment by BusbyGT** [comment]
@winder I did more investigation tonight and the issue is actually the script that I attached in the first post has an error with a G0 command during parsing. The third G38.2 command is never actually sent to GRBL or loaded by UGS. I did figure out the issue with the script though.

The issue is that there is a space in the "G91 G0 Y- 0.625" command. If I remove the space, it loads perfectly fine but I haven't tested sending the script yet to the machine.

I've seemingly always had the space there and it has never caused a problem with GRBL. This must be an artifact of the recent change to the Gcode parsing, Is it a legal command?

![probe-g0-parse-error](https://user-images.githubusercontent.com/26131866/30624425-637b6806-9d8b-11e7-8857-383c73dedfb1.png)

Here's the console when you stream the file anyways:

![probe-console](https://user-images.githubusercontent.com/26131866/30624468-9fa0036e-9d8b-11e7-8486-594e6c8db2de.png)
